js/src/components/run/RunModal.tsx

```diff
@@ -37,12 +52,38 @@ export const RunModal = <PT,>({
 }: RunModalProps<PT>) => {
   const [params, setParams] = useState<Partial<PT>>(defaultParams);
   const [isReadyToExecute, setIsReadyToExecute] = useState(false);
+  const documentationUrl = getDocumentationUrl(type);
 
   return (
     <Modal isOpen={isOpen} onClose={onClose} size="6xl" scrollBehavior="inside">
       <ModalOverlay />
       <ModalContent overflowY="auto" height="75%" minHeight={"400px"}>
-        <ModalHeader>{title}</ModalHeader>
+        <ModalHeader>
+          <Flex alignItems="center" gap={2}>
+            {title}
+            {documentationUrl && (
+              <Tooltip
+                label="Learn more about the SQL behind"
```

Comment:
This message doesn't convey the information that the user can click to view the documentation. Ideally, it would be a triggered popup that would say, "Click here to learn more about the SQL behind this", where "here" is a link to the documentation